test/unittest/StatisticsBackend/CMakeLists.txt

```diff
@@ -279,16 +349,23 @@ add_executable(get_data_tests GetDataTests.cpp
     ${PROJECT_SOURCE_DIR}/src/cpp/topic_types/typesPubSubTypes.cxx
     ${PROJECT_SOURCE_DIR}/src/cpp/types/EntityId.cpp)
 
+if(MSVC)
+    target_compile_definitions(get_data_tests
+        PRIVATE _CRT_DECLARE_NONSTDC_NAMES=0 FASTDDS_STATISTICS_BACKEND_SOURCE)
+endif(MSVC)
+
 target_include_directories(get_data_tests PRIVATE
     ${GTEST_INCLUDE_DIRS} ${GMOCK_INCLUDE_DIRS}
     ${PROJECT_SOURCE_DIR}/include
-    ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME}
+    ${PROJECT_SOURCE_DIR}/include/${PROJECT_NAME_UNDERSCORED}
     ${PROJECT_BINARY_DIR}/include
-    ${PROJECT_BINARY_DIR}/include/${PROJECT_NAME}
+    ${PROJECT_BINARY_DIR}/include/${PROJECT_NAME_UNDERSCORED}
     ${PROJECT_SOURCE_DIR}/src/cpp)
 
 target_link_libraries(get_data_tests PUBLIC
-    ${GTEST_LIBRARIES} ${GMOCK_LIBRARIES} fastrtps)
+    ${GTEST_LIBRARIES} ${GMOCK_LIBRARIES} fastrtps fastcdr)
```

Comment:
Same as before, please check if fastcdr is necessary

Reply:
Any test building `types.cxx` will require fastcdr